doc/source/user_guide/krylov.rst

```diff
@@ -0,0 +1,156 @@
+=======================================
+Harmonic analysis via the krylov method
+=======================================
+
+Introduction
+============
+You can use the frequency-sweep Krylov method for a high performance solution of forced-frequency simulations 
+in acoustic or single-field structural analyses.
+
+Similar to the full harmonic analysis, the frequency sweep Krylov method also uses the full system matrices 
+to compute the harmonic response. While the full method solves the full system matrices at every frequency point 
+in the frequency range, the frequency-sweep Krylov method performs the following steps to approximate the response 
+across the frequency range :
```

Comment:
```suggestion
Similar to the full harmonic analysis, the frequency sweep Krylov method uses full system matrices 
to compute the harmonic response. While the full method solves the full system matrices at every frequency point 
in the frequency range, the frequency sweep Krylov method performs the following steps to approximate the response across the frequency range:
```

Reply:
There is so many occurrences of "full" in the preceding sentences that I feel that readability is reduced. Are they all needed?

Reply:
removed "full" from parts of the sentence to improve readability.